about.html

```diff
@@ -12,53 +12,100 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
-    <img src="img/about-plan.png" alt="strategy">
+    <div class="container about-page">
+
+      <section class="top-content">
+
+        <header>
+          <nav>
+            <div class="logo">
+                <img src="img/lambda-black.png" alt="Lambda">
+            </div>
+            <div class="links">
```

Comment:
The links navigating to index and about.html will not work this way.  The path to get to another page is relative to the file you add the link in.  If you are at index.html and adding a link to about.html it would just be about.html because about.html is in the same folder as index.html